server/planning/planning.py

```diff
@@ -69,13 +80,158 @@ class PlanningService(superdesk.Service):
     },
 
     # Planning Details
-    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16.5.1
+    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16
     # probably can skip this subsection, although its documented in iptc impl guide this way
-    'planning_details': {
+    'planning_item': {
         'type': 'dict',
         'schema': {
-            # TODO: fill in from NewsML-G2 version 2.21 spec
-        }
+            # See IPTC-G2-Implementation_Guide 16.1
+            'item_meta': {
+                'type': 'dict',
+                'schema': {
+                    'item_class': {'type': 'string'}
+                }
+            },
+            # See IPTC-G2-Implementation_Guide 16.2
+            'content_meta': {
+                'type': 'dict',
+                'schema': {
+                    'qcode': {'type': 'string'},
+                    'name': {'type': 'string'}
+                }
+            },
+            # See IPTC-G2-Implementation_Guide 16.3
+            'news_coverage_set': {
```

Comment:
imo this should be an entity `coverage` with `planning_item` reference, there will be lists of coverages filtered by users etc to see assignments, if it's embedded it will be harder to work with it imo